docs/source/cli/graph-construction/single-machine-gconstruct.rst

```diff
@@ -33,7 +33,7 @@ Full argument list of the ``gconstruct.construct_graph`` command
 * **-\-num-processes-for-nodes**: the number of processes to process node data simultaneously. Increase this number can speed up node data processing.
 * **-\-num-processes-for-edges**: the number of processes to process edge data simultaneously. Increase this number can speed up edge data processing.
 * **-\-output-dir**: (**Required**) the path of the output data files.
-* **-\-graph-name**: (**Required**) the name assigned for the graph.
+* **-\-graph-name**: (**Required**) the name assigned for the graph. The graph name must adhere to the Python identifier naming rules with the exception that hyphens (``-``) are permitted.
```

Comment:
Can we give a link of the `Python identifier naming rules` here, or provide some valid and invalid examples?

Reply:
Added